### PR TITLE
Updating release docs to match current process

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: developers/release.md [hunk 1]
```diff
@@ -27,79 +27,32 @@ There are a number of repositories that will need to be tagged and branched thro
...
-
-```
-git checkout master
-```
-
-For any of the gems, update the gem version file and the spec file to X.Y.0 and make sure to reset the release to 1. For example,
+For any of the gems, update the gem version file. For example,
 
 ```
 vim lib/katello/version.rb
```
Comment (johnpmitsch) [inline developers/release.md]: What am I supposed to do here?

File: developers/release.md [hunk 2]
```diff
@@ -27,79 +27,40 @@ There are a number of repositories that will need to be tagged and branched thro
...
-git push origin KATELLO-X.Y
+vim lib/katello/version.rb
 ```
 
-Now we set master to start from the version after the one we are working on releasing X.Y+1, for each package, ensuring you are in the directory the spec file exists:
+Now change the version in the file and save it. For example, if currently working on the release for 3.2, change the version to 3.3.0. Make a commit bumping the version and push it to master.
 
 ```
-git checkout master
+git commit -a -m 'Version bump to X.Y.0'
```
Comment (johnpmitsch) [inline developers/release.md]: Do I 'git push origin Katelllo-3.2' now?

File: developers/release.md [hunk 3]
```diff
@@ -27,79 +27,40 @@ There are a number of repositories that will need to be tagged and branched thro
...
-```
 
 Now for all of the repositories including the gems, tag and push the starting place for the next version:
 
-```
-tito tag --use-version=X.Y+1.0
-git push origin master && git push --tags
-```
-
 For `hammer-cli-katello`:
```
Comment (johnpmitsch) [inline developers/release.md]: Do we still need hammer-cli-katello instructions?

File: developers/release.md [hunk 4]
```diff
@@ -27,78 +27,40 @@ There are a number of repositories that will need to be tagged and branched thro
...
-VER=X.Y sed -i "s#/katello-nightly-rhel/#/katello-$VER-rhel/#g" rel-eng/*
-git commit -a -m 'Updating rel-eng for X.Y'
 git push origin KATELLO-X.Y
 ```
 
-Now we set master to start from the version after the one we are working on releasing X.Y+1, for each package, ensuring you are in the directory the spec file exists:
+Now we set master to start from the version after the one we are working on releasing X.Y+1 For the Katello repositoriy, the gem version file needs updating to the next release. First, open:
 
 ```
-git checkout master
```
Comment (johnpmitsch) [inline developers/release.md]: why did you remove this line? Do I not checkout master?

File: developers/release.md [hunk 5]
```diff
@@ -27,78 +27,40 @@ There are a number of repositories that will need to be tagged and branched thro
...
+vim lib/katello/version.rb
 ```
 
-For any of the gems, update the gem version file and the spec file to X.Y.0 and make sure to reset the release to 1. For example,
+Now change the version in the file and save it. For example, if currently working on the release for 3.2, change the version to 3.3.0. Make a commit bumping the version and push it to master.
 
 ```
-vim lib/katello/version.rb
-vim rubygem-katello.spec
 git commit -a -m 'Version bump to X.Y.0'
```
Comment (johnpmitsch) [inline developers/release.md]: still unsure what I should do after I commit

File: developers/release.md (hunk 4 above)
Comment (ehelms) [inline developers/release.md]: Definitely needs to be there, updated

File: developers/release.md [hunk 6]
```diff
@@ -27,78 +27,42 @@ There are a number of repositories that will need to be tagged and branched thro
...
-  katello-installer.spec
-
-hammer-cli-katello
-  rubygem-hammer_cli_katello.spec
+git clone https://github.com/Katello/katello-packaging
 ```
 
 ### Step 4: Get Your Branch On
 
 On the date and time you previously warned the community branching would occur on, do the following for each repository EXCEPT `hammer-cli-katelo`.
```
Comment (johnpmitsch) [inline developers/release.md]: Should we say 'EXCEPT `hammer-cli-katelo`.' ?

File: developers/release.md [hunk 7]
```diff
@@ -27,78 +27,42 @@ There are a number of repositories that will need to be tagged and branched thro
...
 
 ```
-vim lib/katello/version.rb
-vim rubygem-katello.spec
 git commit -a -m 'Version bump to X.Y.0'
+git push origin master
 ```
 
-Now for all of the repositories including the gems, tag and push the starting place for the next version:
+The same will need to be done for the katello-installer `VERSION` file.
```
Comment (johnpmitsch) [inline developers/release.md]: lib/katello/version.rb doesn't exist in katello-installer right?

If so, can we say something like
> For katello-installer you will modify the `VERSION` file

Also, what do I do for katello-packaging repo?

**Comment by ehelms** [comment]
@jlsherrill can you outline the steps you took initially for katello-packaging? I'll write it up but since you did it most recently, your recollection will be the freshest.

**Comment by jlsherrill** [comment]
Create a KATELLO-X.Y branch in katello-packaging

In master of all these other repos, bump their version to X.Y+1
katello/katello
katello/katello-installer

In master on katello-packaging, bump the version to X.Y+1 for these packages in the rpm spec:
rubygem-katello
katello-installer
repos
katello

At RC1 build time:

In KATELLO-X.Y on these repos:

rubygem-katello
katello-installer

bump their version to X.Y.rc1.  Build the rubygem-katello and push it to rubygems

In KATELLO-X.Y in katello-packaging, set prever to '.rc1' on:
rubygem-katello
katello-installer
repos
katello

and build all as normal.

I *think* that is it, might have missed something, but its a good start

**Comment by johnpmitsch** [comment]
@ehelms this should be good once you add the step for katello-packaging that @jlsherrill detailed

```
In master on katello-packaging, bump the version to X.Y+1 for these packages in the rpm spec:
rubygem-katello
katello-installer
repos
katello
```
Let me know if you want me to write something up since I just went through it
